samples/winapi_drivelist/main.c

```diff
@@ -9,45 +9,47 @@ int main(void)
 
     // Mount some drives for demonstration purposes
     BOOL ret;
+    DWORD error;
     ret = nxMountDrive('C', "\\Device\\Harddisk0\\Partition2\\");
     if (!ret) {
-        debugPrint("Failed to mount C: drive!\n");
+        // Additional error info can be retrieved with GetLastError()
+        error = GetLastError();
+        debugPrint("Failed to mount C: drive! Error code: %x\n", error);
         Sleep(5000);
         return 1;
     }
+
     ret = nxMountDrive('E', "\\Device\\Harddisk0\\Partition1\\");
     if (!ret) {
-        debugPrint("Failed to mount E: drive!\n");
-        Sleep(5000);
-        return 1;
+        error = GetLastError();
+        debugPrint("Failed to mount E: drive! Error code: %x\n", error);
+        goto unmount_c;
     }
 
-    // Retrieve drive bitmaks. Every bit represents one drive letter
+    // Retrieve drive bitmasks. Every bit represents one drive letter
     DWORD driveBits = GetLogicalDrives();
     if (driveBits == 0 && GetLastError() != ERROR_SUCCESS) {
-        debugPrint("Failed to retrieve drive bitmask!\n");
-        Sleep(5000);
-        return 1;
+        error = GetLastError();
```

Comment:
Doing `GetLastError()` twice feels like bad practice that shouldn't appear in samples.